### PR TITLE
Add dynamic ASG overview doc

### PR DESCRIPTION
**Pull request by ameowlia**

### CODE REVIEW

File: docs/dynamic-asg-overview.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+| job name | property name | default | link to code|
+| --- | --- | --- | --- |
+| policy-server-asg-syncer | asg_poll_interval_seconds | 60 | [link](https://github.com/cloudfoundry/cf-networking-release/blob/0c5029c88e9f61bb94829b4e0b8ed6732f30f9f0/jobs/policy-server-asg-syncer/spec#L31-L33) |
+| vxlan-policy-agent | asg_poll_interval_seconds | 60 | [link](https://github.com/cloudfoundry/silk-release/blob/f1606499925ff94bc036a641d688967c7af1fef4/jobs/vxlan-policy-agent/spec#L59-L61) |
+
+### 5. Did we stop passing ASG information through the LRP?
+No. ASGs are still passed through Cloud Controller to BBS to Rep to Garden to
+Silk CNI. This allows ASGs to be implemented correctly the minute an app is
+created.
```
Comment (geofffranks) [inline docs/dynamic-asg-overview.md]: We call a force-asg-sync endpoint from the CNI, which overrides any of the data passed from garden to the CNI.

However, we still pass data through Diego to Garden for non-silk CNIs, so they keep their existing behavior.

Comment (geofffranks) [reply docs/dynamic-asg-overview.md]: The diagram + steps for applying rules should maybe be updated for this force-sync as well?

**Review by geofffranks — CHANGES_REQUESTED** [review]
Overall looks great! Left a note about the container start process under dynamic-asgs that should get addressed though

File: docs/dynamic-asg-overview.md (hunk 1 above)
Comment (ameowlia) [reply docs/dynamic-asg-overview.md]: I decided not to edit the diagram to include the force-sync. I thought adding the container create flow made it too cluttered. I did add it in one of the FAQs though.

**Review by geofffranks — APPROVED** [review]

**Pull request merged**